prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -61,8 +67,24 @@ abstract class HistoryDatabaseAccess(
                 r?.into(DetailedActionLog::class.java)
             },
             DSL.multiset(
-                DSL.select()
+                DSL.select(
+                    REPORT_FILE.REPORT_ID, REPORT_FILE.RECEIVING_ORG, REPORT_FILE.RECEIVING_ORG_SVC,
+                    REPORT_FILE.SENDING_ORG, REPORT_FILE.SENDING_ORG_CLIENT, REPORT_FILE.SCHEMA_TOPIC,
+                    REPORT_FILE.EXTERNAL_NAME, REPORT_FILE.CREATED_AT, REPORT_FILE.NEXT_ACTION_AT,
+                    REPORT_FILE.ITEM_COUNT, REPORT_FILE.ITEM_COUNT_BEFORE_QUAL_FILTER,
+                    field("\"receiver\".\"values\"->>'transport' IS NOT NULL")
+                        .`as`("receiverHasTransport")
+                )
                     .from(REPORT_FILE)
+                    // Not ideal but to return the sender so that we can get the sending org and
```

Comment:
This comment is not needed - this is ideal, since there are 0...1 receivers specified on a report, so using a left join is appropriate.